### PR TITLE
Refactor example code

### PR DESCRIPTION
**Pull request by SenorGato**
As seen here -> https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onMessage#sending_a_synchronous_response  

The preceding code did not work, and the code from the async does.  Its also cleaner.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md [hunk 1]
```diff
@@ -111,11 +111,12 @@ The corresponding background script looks like this:
...
 ```js
 // background-script.js
+Response({response: "Response from background script"});
+}
 
 function handleMessage(request, sender, sendResponse) {
-  console.log("Message from the content script: " +
-    request.greeting);
-  sendResponse({response: "Response from background script"});
+  console.log(`content script sent a message: ${request.content}`);
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md:118]:
```suggestion
  console.log(`A content script sent a message: ${request.content}`);
```

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md [hunk 2]
```diff
@@ -111,11 +111,12 @@ The corresponding background script looks like this:
...
+  sendResponse({response: "response from background script"});
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md:119]:
```suggestion
  sendResponse({response: "Response from background script"});
```

**Review by rebloor — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md [hunk 3]
```diff
@@ -111,10 +111,11 @@ The corresponding background script looks like this:
 
 ```js
 // background-script.js
+Response({response: "Response from background script"});
+}
```
Comment (rpl) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md:115]: This looks like a left over, it doesn't seem to belong in the code snippet.

File: files/en-us/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md [hunk 4]
```diff
@@ -111,10 +111,11 @@ The corresponding background script looks like this:
...
 
 function handleMessage(request, sender, sendResponse) {
-  console.log("Message from the content script: " +
-    request.greeting);
+  console.log(`A content script sent a message: ${request.content}`);
```
Comment (rpl) [inline files/en-us/mozilla/add-ons/webextensions/api/runtime/sendmessage/index.md:118]: at line 102, the content script seems to be sending an object with a `"greeting"` property (and not a property named `"content"`).

**Review by rpl — CHANGES_REQUESTED** [review]

**Review by SenorGato — COMMENTED** [review]
This seems to make sense to me.

**Review by rebloor — APPROVED** [review]

**Review by rpl — APPROVED** [review]
lgtm, besides a few formatting related tweaks and a `let` changed into a `const` the new version is equivalent to the previous version.

**Pull request merged**